docs/installation.rst

```diff
@@ -139,4 +178,25 @@ Usually is useful to install examples along with the XRCE-DDS suite, for doing s
 
 .. note::
     If you want to install the *eProsima Micro XRCE-DDS* examples, you can add :code:`-DUXRCE_BUILD_EXAMPLES=ON`
-    to the :code:`cmake ..` command-line options. This flag will enable the compilation of the examples.
\ No newline at end of file
+    to the :code:`cmake ..` command-line options. This flag will enable the compilation of the examples.
+
+.. _install_agent_client_docker:
+
+Using the provided Docker image
+*******************************
+
+    *eProsima Micro XRCE-DDS* is also available as a whole package in a `Docker image. <https://hub.docker.com/r/eprosima/micro-xrce-dds-agent>`_
+
+    Within this Docker image, the *Micro XRCE-DDS Agent* standalone application and library are installed, as well as the *Micro XRCE-DDS Client* library and built-in examples.
```

Comment:
Within -> With

Reply:
well, actually the installation is done inside the Docker. Wouldn't it make more sense to keep it as within?